src/main/webapp/app/exam/exam-scores/exam-scores-average-scores-graph.component.html

```diff
@@ -1,5 +1,24 @@
 <div class="row">
-    <div class="col-md-7">
-        <canvas baseChart [height]="height" [datasets]="chartData" [labels]="barChartLabels" [options]="barChartOptions" [legend]="chartLegend" [chartType]="barChartType"></canvas>
+    <div #containerRef class="col-md-7" style="align-items: center">
```

Comment:
This inline style can also be replaced with bootstrap (maybe [this](https://bootstrapshuffle.com/de/classes/flexbox/align-items-*-center) will help)

Reply:
Done in ddade2e